### DISCUSSION TITLE
Parametros no topo do método

### DISCUSSION THREAD

**Discussion by Rafael354-dev** [balta-io/2811#10] [Q&A] [answered]
Parâmetros do método não aparecem igual ao do curso.

[HttpGet("v1/categories/{id}")] // Id esta como se fosse um texto normal
        public async Task<IActionResult> GetByIdAsync([FromRoute] int id, [FromServices] BlogDataContext context)
        {
            var category = await context.Categories.FirstOrDefaultAsync(x => x.Id == id);

            if (category == null)
                return NotFound();

            return Ok(category);
        }

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Rafael354-dev , como vai?

Não entendi sua questão! Qual erro dá? O que está errado? Poderia detalhar melhor, por favor?

**Comment by Rafael354-dev** [comment]
Olá Balta, vou enviar um print pra vc ver.

***@***.***


Repare q o dentro de Route não ficou igual mostrado no curso.

Att.

Rafael
Enviado do Email<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: Andre ***@***.***>
Enviado:sexta-feira, 11 de fevereiro de 2022 09:41
Para: ***@***.***>
Cc:Rafael Gianini ***@***.***>; ***@***.***>
Assunto: Re: [balta-io/2811] Parametros no topo do método (Discussion #10)


Bom dia, @Rafael354-dev<https://github.com/Rafael354-dev> , como vai?

Não entendi sua questão! Qual erro dá? O que está errado? Poderia detalhar melhor, por favor?

—
Reply to this email directly, view it on GitHub<https://github.com/balta-io/2811/discussions/10#discussioncomment-2157014>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATYS5UMUMNOSKTKIRSKV5UDU2T7WZANCNFSM5OCZRAGA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by andrebaltieri** [reply]
Não veio a imagem, acho que você respondeu o E-mail do GitHub, aí ele não anexa!
